soepy/python/solve/solve_auxiliary.py

```diff
@@ -180,8 +185,9 @@ def pyth_backward_induction(attr_dict, state_space_args):
         for k in range(states_number_period[period]):
 
             # Construct additional education information
-            educ_level, educ_years_idx = construct_covariates(
-                attr_dict, states_all, period, k
+            educ_level, educ_years_idx = (
```

Comment:
Why not just 

`educ_level, educ_years_idx = covariates[period, k, 0:3], covariates[period, k, 3]`

on one line. Why create the tuple?